io.github.daviddesimone.opencloudsaves.yml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.daviddesimone.opencloudsaves
+base: io.elementary.BaseApp
+base-version: juno-21.08
```

Comment:
Are you sure you need this? Elementary BaseApp is unmaintained, and it doesn't match the runtime/SDK you're using.

Reply:
Good catch - thank you. This is corrected.